src/plugins/intel_cpu/src/emitters/plugin/x64/utils.cpp

```diff
@@ -37,7 +37,7 @@ struct regs_to_spill {
     static std::vector<Xbyak::Reg> get(const std::set<snippets::Reg>& live_regs) {
         std::vector<Xbyak::Reg> regs_to_spill;
         auto push_if_live = [&live_regs, &regs_to_spill](Xbyak::Reg&& reg) {
-            if (live_regs.empty() || live_regs.count(Xbyak2SnippetsReg(reg))) {
+            if (live_regs.empty() || (live_regs.count(Xbyak2SnippetsReg(reg)) != 0u)) {
```

Comment:
Please double check the changes with respect to the relaxed requirements.
The best way probably would be to reapply them from scratch.

Reply:
Re-applied it from scratch. It seems this is covered in any case.